lms/models/json_settings.py

```diff
@@ -103,7 +110,19 @@ def get(self, group: str, key: str, default=None):  # type: ignore[override]
         return super().get(group, {}).get(key, default)
 
     def get_setting(self, setting: JSONSetting):
-        return self.get(setting.group, setting.key, setting.default)
+        value = self.get(setting.group, setting.key, setting.default)
+        if value is None:  # None is used a sentinel for "unset" or "default"
```

Comment:
We can get None when the value was never set and when it was explicitly set to None. Both should result on the default value.